dev/webpack/webpack.dev.js

```diff
@@ -35,6 +35,7 @@ module.exports = {
     pathinfo: true,
     crossOriginLoading: 'use-credentials'
   },
+  devtool: 'cheap-module-eval-source-map',
```

Comment:
Submit a different PR for this and make it optional, as it slows down the build.

Reply:
maybe just merge the commit without that change? 21 days is a bit long for such a trivial change and I was running into this issue as well and though the setup just wasn't working at all...